### DISCUSSION TITLE
Possible to open Save dialog during exam

### DISCUSSION THREAD

**Discussion by uskomatonta** [SafeExamBrowser/seb-mac#374] [General]
We are using SEB and settings via Moodle. At least on Mac version it seems to be possible to hit command S and open save dialog during exam. This could be used to create folders with answers to questions. Check the answer from this folder list and click cancel to go back to exam and answer the question. Not tested with Windows.

**Comment by danschlet** [comment]
Can you please provide full information to reproduce this issue? See template when creating a [bug report issue](https://github.com/SafeExamBrowser/seb-mac/issues/new?assignees=danschlet&labels=&projects=&template=bug_report.md&title=).

1. Is this happening with an essay question type in Moodle when you can upload files? If I just start Moodle with SEB 3.3.2 (for example with this link: [Demo Exams](https://safeexambrowser.org/exams/), take the first one
SEB Demo Exam, quit password "quit"), nothing happens when hitting command-S when the Moodle quiz is running. 
2. Or is this happening when using some other (custom?) Moodle question type? If yes, which one? Please provide a demo/testing account so we can try that out. 
3. If it happens when when Moodle asks to upload files: There are options in SEB's [Down/Upload](https://safeexambrowser.org/macosx/mac_usermanual_en.html#DownUploadsPane) settings (manual not yet updated for SEB 3.3.2). If you change the secure default to "Allow downloading and uploading files" and "Allow uploading files", and leave the setting "Choose file to upload …" on the default "manually with file requester", then the save dialog can be displayed when files are uploaded. You can currently choose the option "by only allowing to upload the same file downloaded before", which is the option which won't display a save file dialog.

**Comment by danschlet** [comment]
Can you please provide full information to reproduce this issue? See template when creating a [bug report issue](https://github.com/SafeExamBrowser/seb-mac/issues/new?assignees=danschlet&labels=&projects=&template=bug_report.md&title=).

1. Is this happening with an essay question type in Moodle when you can upload files? If I just start Moodle with SEB 3.3.2 (for example with this link: [Demo Exam](sebs://safeexambrowser.org/exams/DemoExamGeneral.seb), quit password "quit"), nothing happens when hitting command-S when the Moodle quiz is running.
2. If it happens when when Moodle asks to upload files: There are options in SEB's [Down/Upload](https://safeexambrowser.org/macosx/mac_usermanual_en.html#DownUploadsPane) settings. If you change the secure default to "Allow uploads"

**Comment by danschlet** [comment]
See issue #382.
